weDevsOfficial/dokan
# Feat / Dokan dummy data

### All Submissions:






### Changes proposed in this Pull Request:

This feature will allow admins to create dummy vendors and some products for vendors by clicking a single button. And also can remove those vendors and products by clicking one single button.

### How to test the changes in this Pull Request:

1. Build dokan lite.
2. There is a new menu Import dummy data in dokan dashboard get help question mark icon [Screenshot](https://www.awesomescreenshot.com/image/27448193?key=f62f7aff189b75bf6bd6ebf8867aae04)
3. Clicking Import dummy data a new page will appear there is a button called to run the importer clicking this it will import dummy vendors and products. [Screenshot](https://tinyurl.com/y4gho26p)
4. After completing the import it will show a success message and you can also clear all the dummy data by clicking clear dummy data. [Screenshot](urhttps://tinyurl.com/y3uro5nel)



### Changelog entry

> Feature: Dokan dummy data.


### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.


### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
